src/mscorlib/shared/System/Version.cs

```diff
@@ -211,8 +211,10 @@ public bool TryFormat(Span<char> destination, int fieldCount, out int charsWritt
                 charsWritten = 0;
                 return true;
             }
-
-            // TODO https://github.com/dotnet/corefx/issues/22403: fieldCount==1 can just use int.TryFormat
```

Comment:
Oops; I neglected to search for the issue number.  Thanks for cleaning up after me.